src/modules/verifiable-credentials/verifiable-credentials-base.service.ts

```diff
@@ -108,7 +111,12 @@ export abstract class VerifiableCredentialsServiceBase {
    * @description The type of the exchange. Only vc-api exchanges currently supported.
    * @returns credentials query with matching verifiable presentations
    */
-  async initiateExchange({ type, url }: ExchangeInvitation) {
+  async initiateExchange({ type, url }: ExchangeInvitation): Promise<
+    {
+      selectResults: SelectResults;
+      presentationDefinition: PresentationDefinition;
+    }[]
```

Comment:
Nice. Personally, I'm a fan of having the return type on the method signature

Reply:
I am also not a fan of inline types